### PR TITLE
Fixes #33288 - Enter key fix for content view filter modal

### PR DESCRIPTION
**Pull request by Andrewgdewar**

### CODE REVIEW

File: webpack/scenes/ContentViews/Details/Filters/Add/CVFilterAddModal.js [hunk 1]
```diff
@@ -134,12 +134,12 @@ const CVFilterAddModal = ({ cvId, show, setIsOpen }) => {
           <Button
             aria-label="create_filter"
             variant="primary"
-            isDisabled={saving}
-            onClick={() => onSave()}
+            isDisabled={saving || name.length === 0}
+            type="submit"
```
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Details/Filters/Add/CVFilterAddModal.js:138]: :tada:

**Review by jeremylenz — APPROVED** [review]
Thanks for fixing this @Andrewgdewar!

Looks like it's broken a couple tests though.

APJ

**Review by sjha4 — APPROVED** [review]

**Comment by sjha4** [comment]
[test katello]

**Comment by sjha4** [comment]
[test katello]

**Pull request merged**